Documents/admin-api.md

```diff
@@ -173,3 +173,136 @@ This API is for internal use only. The following endpoints are available
     }
     ```
 
+* `/apor`
+
+    * `POST`
+
+    Creates a new APOR value
+
+    Example request, in `JSON` format
+
+    ```json
+        {
+        	"rateType": "FixedRate",
+        	"newApor": {
+        		"rateDate": "2018-01-01",
+        		"values": [
+        			1.00,
+        			3.26,
+        			3.39,
+        			3.39,
+        			3.59,
+        			3.59,
+        			3.68,
+        			3.68,
+        			3.74,
+        			3.74,
+        			3.74,
+        			3.74,
+        			3.51,
+        			3.51,
+        			3.51,
+        			3.51,
+        			3.51,
+        			3.51,
+        			3.51,
+        			3.51,
+        			3.51,
+        			3.51,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03,
+        			4.03
+        		]
+        	}
+        }
+        ```
+
+    If a value for that date already exists, this new entry will not be created (see below for endpoint to update values).
```

Comment:
From my tests, I think this line of documentation isn't accurate: "If a value for that date already exists, this new entry will not be created". In my testing, I've seen it create a new APOR record each time there's a `POST`, even if the new record conflicts with existing ones. 

If it **should** be rejecting any conflicting requests, we'll need to update line 116 of `HmdaAPORPersistence.scala` to check whether the date + amortization combo already exists in memory.

If we do that, we should remove this bit, since it won't be accurate anymore: "If an APOR record already exists in the system with the given rate type and date, the new record will effectively overwrite the existing one. Both will exist in the system, but the most recently added one will be used in rate spread calculations"

Reply:
I'm removing that line in the documentation and not changing the behavior. The APOR values are in general not going to be inserted through this mechanism (they will be ingested through S3). This method is just a fallback in case that fails.